medical-portal/src/API/RSBC.DMF.MedicalPortal.API/Controllers/ChefsController.cs

```diff
@@ -217,6 +217,7 @@ public async Task<ActionResult> GetChefsBundle([Required] [FromQuery] string cas
             {
                 chefsBundle.patientCase = caseResult;
                 caseResult.DriverLicenseNumber = c.Item.DriverLicenseNumber;
+                chefsBundle.medicalConditions = c.Item.MedicalConditions;
```

Comment:
after making the below change, you will need to map this from proto model to view model

Reply:
Done, thanks!